content/inside-rust/stage0-redesign/index.md

```diff
@@ -0,0 +1,126 @@
++++
+path = "inside-rust/9999/12/31/redesigning-the-initial-bootstrap-sequence"
+title = "Redesigning the Initial Bootstrap Sequence"
+authors = ["Jieyou Xu"]
+
+[extra]
+team = "the Bootstrap team"
+team_url = "https://www.rust-lang.org/governance/teams/infra#team-bootstrap"
++++
+
+This blog post accompanies an [upcoming major change to the `bootstrap` build system][stage0-redesign-pr] (see also [Major Change Proposal 619][redesign-stage0-mcp]), which is expected to affect contributors who works on the `rustc` compiler, the standard library, tools, and the distributors of rust toolchains.
```

Comment:
```suggestion
This blog post accompanies an [upcoming major change to the rust-lang/rust build system][stage0-redesign-pr] (see also [Major Change Proposal 619][redesign-stage0-mcp]). This will have no impact on the distributed artifacts from that repository, but the way we build those artifacts is changing.
```